targets/targets.json

```diff
@@ -4217,5 +4217,19 @@
         "inherits": ["FVP_MPS2"],
         "core": "Cortex-M7",
         "macros": ["CMSDK_CM7"]
+    },
+    "RDA5981X": {
+        "inherits": ["Target"],
+        "core": "Cortex-M4F",
+        "public": false,
```

Comment:
Why this is false at the moment (plan to have it public?)

Reply:
changed it to true